config/testgrids/conformance/conformance-all.yaml

```diff
@@ -104,28 +96,6 @@ dashboards:
 
 - name: conformance-apisnoop
 - name: conformance-gce
-- name: conformance-cloud-provider-openstack
```

Comment:
FWIW I have found it useful to see conformance results aggregated in this one dashboard group (e.g. https://testgrid.k8s.io/conformance) but we can revisit later